src/Sylius/Component/Core/spec/OrderProcessing/OrderTaxesProcessorSpec.php

```diff
@@ -137,4 +161,35 @@ function it_does_not_process_taxes_if_there_is_no_tax_zone(
 
         $this->process($order);
     }
+
+    function it_does_not_process_taxes_if_there_is_no_customer_tax_category(
```

Comment:
Is it a valid assumption? It would be safer to throw an exception than to make someone pay no taxes in our shop because of misconfigured taxes.

Reply:
IMO the same case applies to tax zones, I wanted to implement this concept similar to tax zone. What do you say to discuss this later and apply proper behaviour in a separate PR for both concepts?